kconfiglib.py

```diff
@@ -1445,19 +1445,32 @@ def write_autoconf(self, filename=None, header=None):
           will be used if it was set, and no header otherwise. See the
           Kconfig.header_header attribute.
 
+        negatives (default: None):
+          If True, this will write #defines for false values as well,
+          defined to the value 0.
+
+          If False, #defines for false values will not be written.
+
+          If None (the default), this will be influenced by the
+          KCONFIG_NEGATIVES environment variable, just like in some C
+          implementations of Kconfig.
+
         Returns a string with a message saying that the header got saved, or
         that there were no changes to it. This is meant to reduce boilerplate
         in tools, which can do e.g. print(kconf.write_autoconf()).
         """
         if filename is None:
             filename = os.getenv("KCONFIG_AUTOHEADER",
                                  "include/generated/autoconf.h")
+        if negatives is None:
+            negatives = "KCONFIG_NEGATIVES" in os.environ
```

Comment:
Should we check if `KCONFIG_NEGATIVES=1`?

Reply:
Sure, technically the C impl just checks if present:

```
if (getenv("KCONFIG_NEGATIVES")) { ...
```

But since this wouldn't break coreboot/depthcharge behavior to check for `1`, seems like a more intuitive choice.